app/styles/modules/_settings.scss

```diff
@@ -688,19 +698,24 @@ section.modal-panel {
   white-space: nowrap;
   width: calc(95% - 95px);
 
-  & .not-verified {
-    color: $color-red;
+  &.verified {
+    // There are two buttons when the email
+    // is verified, the extra button width
+    // needs to be taken into account.
+    // See #6751
+    width: calc(95% - 195px);
+    color: $color-green;
   }
 
-  & .verified {
-    color: $color-green;
+  &.not-verified {
+    color: $color-red;
   }
 
-  & .disabled {
```

Comment:
I don't think the `&` in these selectors had any effect. I looked at the templates to see how they were used and it seems like they were supposed to be child selectors.